bundles/net.tourbook/src/net/tourbook/map25/Map25View.java

```diff
@@ -231,22 +232,52 @@
    private Bitmap                        _bitmapPhoto;
    private Bitmap                        _bitmapStar;
    //final Paint                               _fillPainter = CanvasAdapter.newPaint();
-
    //
    private int     _leftSliderValueIndex;
    private int     _rightSliderValueIndex;
    private int     _selectedSliderValueIndex;
    //
    private int     _hash_AllPhotos;
-   private int     _hashTourId;
+private int     _hashTourId;
+
    private int     _hashTourData;
+
    //
    private MapSync _mapSynchedWith = MapSync.NONE;
+
    private long    _lastFiredSyncEventTime;
+
    //
    // context menu
    private boolean _isContextMenuVisible;
-//	private MouseAdapter							_wwMouseListener;
+
+/*
+ * private class ActionShowPhoto_WithConfig extends ActionToolbarSlideout {
+ * public ActionShowPhoto_WithConfig() {
+ * super(
+ * TourbookPlugin.getImageDescriptor(Map25View.IMAGE_ACTION_SHOW_PHOTO_IN_MAP),
+ * TourbookPlugin.getImageDescriptor(Map25View.IMAGE_ACTION_SHOW_PHOTO_IN_MAP_DISABLED));
+ * isToggleAction = true;
+ * notSelectedTooltip = MAP_ACTION_SHOW_PHOTO_IN_MAP;
+ * }
+ * @Override
+ * protected ToolbarSlideout createSlideout(final ToolBar toolbar) {
```

Comment:
Comments which are outside of a method are moved around by the code formatter, but not always.